scripts/third_party_size_check.py

```diff
@@ -21,35 +21,54 @@
 
 import os
 import sys
+import yaml
 
 THIRD_PARTY_PATH = os.path.join(os.getcwd(), 'third_party')
 THIRD_PARTY_SIZE_LIMIT = 7000
 
 
-def _check_size_in_dir(dir_path):
+def _get_skip_files_list():
+    """This lists the directories which should be skipped."""
```

Comment:
Change this docstring description to:
```This function returns the list of the files which are skipped when Oppia is deployed to GAE.```

Also, follow the docstring pattern -- specify `Returns` section as well. See other docstrings in this file.

Reply:
Done Thanks!